benchmarks/request/rpcrequest-payload.js

```diff
@@ -16,8 +16,9 @@ function main({ n, size }) {
   const buf = Buffer.alloc(size);
   buf.fill('x');
 
-  const request = new Request('INSERT INTO #benchmark ([value]) VALUES (@value)', () => {});
+  const request = new Request('INSERT INTO #benchmark ([value], [val]) VALUES (@value, @val)', () => {});
   request.addParameter('value', TYPES.VarBinary, buf);
+  request.addParameter('val', TYPES.VarChar, 'Hello World');
```

Comment:
I think it would be better to split this up into separate benchmark runs. That way running the benchmarks will give numbers that can easier be understood.

We'd have to change the `createBenchmark` call to this:

```js
const bench = createBenchmark(main, {
  n: [10, 100],
  type: ['varbinary', 'varchar']
  size: [
    1024 * 1024,
    10 * 1024 * 1024,
    50 * 1024 * 1024
  ]
});
```

and `main` would need to be updated to add the `type`:

```js
function main({ n, type, size }) {
```

and then the code here on these lines could be changed like this:

```js
  const request = new Request('INSERT INTO #benchmark ([value]) VALUES (@value)', () => {});
  switch (type) {
    case 'varchar': {
      const value = 'x'.repeat(size);
      request.addParameter('value', TYPES.VarChar, value);
      break;
    }

    case 'varbinary': {
      const value = Buffer.alloc(size);
      value.fill('x');
      request.addParameter('value', TYPES.VarBinary, value);
      break;
    }

    default:
      throw new Error('unknown type ${type}');
  }
```